### PR TITLE
Adding the option to add custom environment variables

### PR DESCRIPTION
**Pull request by joshdvir**
I use this to reveal things like build number, git sha and things like that

### CODE REVIEW

**Comment by lbeder** [comment]
Thanks for the PR. Looks like a very nice addition to the gem! :+1: 

Other than very insignificant notes (which I'll leave in the PR in a moment) - looks fantastic. Also, thanks for fixing the new Rubocop warning in the original code!

File: README.md [hunk 1]
```diff
@@ -122,6 +122,18 @@ HealthMonitor.configure do |config|
 end
 ```
 
+### Adding environmet variables
+By default, environmet variables is nil, you need to provide a Hash with your custom environmet variables:
+
+```ruby
+HealthMonitor.configure do |config|
+  config.environmet_variables = {
+    build_number: "BUILD_NUMBER",
```
Comment (lbeder) [inline README.md]: Could you please use single quotes here as well?

File: lib/health_monitor/providers/sidekiq.rb [hunk 2]
```diff
@@ -7,7 +7,7 @@ class SidekiqException < StandardError; end
 
     class Sidekiq < Base
       class Configuration
-        DEFAULT_LATENCY_TIMEOUT = 30.freeze
+        DEFAULT_LATENCY_TIMEOUT = 30
```
Comment (lbeder) [inline lib/health_monitor/providers/sidekiq.rb]: Why did you unfreeze it?

File: spec/controllers/health_monitor/health_controller_spec.rb [hunk 3]
```diff
@@ -64,10 +65,49 @@
     end
   end
 
+  describe 'environmet variables' do
```
Comment (lbeder) [inline spec/controllers/health_monitor/health_controller_spec.rb]: Thanks for adding tests! :+1:

File: lib/health_monitor/providers/sidekiq.rb (hunk 2 above)
Comment (joshdvir) [inline lib/health_monitor/providers/sidekiq.rb]: rubocop said there is no point of freezing

**Comment by joshdvir** [comment]
no worries, this is a great gem i'm enjoying it in production right now.

File: spec/controllers/health_monitor/health_controller_spec.rb (hunk 3 above)
Comment (joshdvir) [inline spec/controllers/health_monitor/health_controller_spec.rb]: :+1:

**Comment by lbeder** [comment]
Let me see if the unf issue on ruby-head can be worked around.

**Comment by lbeder** [comment]
Seems to be that the current, most popular, solution is to ignore ruby-head failures for now (as it's using 2.4.0dev).

I'll merge, perform the fix on the original master, and then release a new version. 

Thanks!

**Pull request merged**